Update debug-toolbars.md (#33)
mezzio/mezzio: docs/book/v3/cookbook/debug-toolbars.md

<?php

use PhpMiddleware\PhpDebugBar\ConfigProvider;
use Psr\Container\ContainerInterface;

return array_merge(ConfigProvider::getConfig(), [
    'dependencies' => [
        'factories' => [
            \PhpMiddleware\PhpDebugBar\PhpDebugBarMiddleware::class => \PhpMiddleware\PhpDebugBar\PhpDebugBarMiddlewareFactory::class,
            \DebugBar\DataCollector\ConfigCollector::class => \PhpMiddleware\PhpDebugBar\ConfigCollectorFactory::class,
            \PhpMiddleware\PhpDebugBar\ConfigProvider::class => function(ContainerInterface $container) {
                return $container->get('config');
            },
            \DebugBar\DebugBar::class => \PhpMiddleware\PhpDebugBar\StandardDebugBarFactory::class,

>             \DebugBar\JavascriptRenderer::class => \PhpMiddleware\PhpDebugBar\JavascriptRendererFactory::class,

Reviewer: To shorten these lines I suggest importing each of the classes, or the namespaces `PhpMiddleware\PhpDebugBar` and `DebugBar`, at the top of the file.

        ]
    ]
]);
```

> In addition, ensure these interfaces are registered as aliases in your container. For example, with Laminas Diactoros:

Reviewer: ```suggestion
In addition, ensure the [PSR-15 HTTP message factory interfaces](https://www.php-fig.org/psr/psr-15/) are registered in your container. For example, when using [Diactoros](https://docs.laminas.dev/laminas-diactoros) as your [PSR-7 HTTP message interfaces](https://www.php-fig.org/psr/psr-7) implementation, you can define the following:
```

```php
return [
    'dependencies' => [

>         'aliases' => [

Reviewer: ```suggestion
        'invokables' => [
```

Needs to be invokables for now, as:

- Diactoros does not register any services currently
- The container needs to know what to instantiate

Alternately, define aliases + factory configuration.

            Psr\Http\Message\ResponseFactoryInterface::class => Laminas\Diactoros\ResponseFactory::class,
            Psr\Http\Message\StreamFactoryInterface::class => Laminas\Diactoros\StreamFactory::class
        ],
        ...
];
```

> Finally, add the `PhpDebugBarMiddleware` class to the pipeline in `config/pipeline.php` after the `ErrorHandler` class:

Reviewer: ```suggestion
Finally, add the `PhpDebugBarMiddleware` class to the pipeline in `config/pipeline.php` after piping the `ErrorHandler` class:
```

```php

> if (!empty($container->get('config')['debug'])) {

Reviewer: ```suggestion
if (! empty($container->get('config')['debug'])) {
```

Examples should follow our CS, which means a space trailing the `!`.

    $app->pipe(PhpDebugBarMiddleware::class);
}
```

> ## Usage in a Request Handler

Reviewer: ```suggestion
### Usage in a Request Handler
```

One additional header level, please
